tests/test_e2e.py

```diff
@@ -181,26 +211,48 @@ def test_equations_to_amr(context_dir, http_mock, client, worker, file_storage):
     job_id = results.get("id")
     worker.work(burst=True)
     status_response = client.get(f"/status/{job_id}")
-    
+
     job = Job.fetch(job_id, connection=worker.connection)
-    if job.result is not None: amr_instance = AMR(job.result["amr"])
+    if job.result is not None:
+        amr_instance = AMR(job.result["amr"])
+
+    # Tests the equation to AMR with a predefined model_id.
+    response_config = client.post(
+        "/equations_to_amr",
+        params=query_params_config_test,
+        data=equations,
+        headers={"Content-Type": "application/json"},
+    )
+    results_config_test = response_config.json()
+    job_id_config = results_config_test.get("id")
+    status_response_config = client.get(f"/status/{job_id_config}")
 
     #### ASSERT ####
     assert results.get("status") == "queued"
     assert status_response.status_code == 200
-    assert status_response.json().get("status") == "finished", f"The RQ job failed.\n{job.latest_result().exc_string}"
+    assert (
+        status_response.json().get("status") == "finished"
+    ), f"The RQ job failed.\n{job.latest_result().exc_string}"
 
     assert (
-            amr_instance.is_valid()
+        amr_instance.is_valid()
     ), f"AMR failed to validate to its provided schema: {amr_instance.validation_error}"
 
+    # Assertions for predefined model id.
```

Comment:
How does this assert the predefined model id?

Reply:
Addressed in the newest commit with assertions to test the configurations.